packages/server/email/notificationSummaryCreator.tsx

```diff
@@ -27,19 +29,39 @@ ${ExternalLinks.TEAM}
 `
 }
 
-const notificationSummaryCreator = (props: NotificationSummaryProps) => {
+interface Props {
+  appOrigin: string
+  preferredName: string
+  notificationCount: number
+  environment: ServerEnvironment
+}
+
+const notificationSummaryCreator = async (props: Props) => {
   const {notificationCount} = props
   const subject = `You have ${notificationCount} new ${plural(
     notificationCount,
     'notification'
   )} 👀`
+
+  const NotificationSummaryEmailRoot =
+    require('parabol-client/modules/email/components/NotificationSummaryEmailRoot').default
```

Comment:
+1 I'm not sure I can follow why you are requiring this here when we import from 'NotificationSummaryEmail' above.

Reply:
This is due to some weirdness at build time. For whatever reason, when this is imported normally, we hit a `MODULE_NOT_FOUND` error on a generated relay file in a build step that happens before we run the relay compiler.

Added a comment + TODO.